src/server/game/DataStores/DB2Structure.h

```diff
@@ -2546,11 +2546,11 @@ struct SpellXSpellVisualEntry
 struct SummonPropertiesEntry
 {
     uint32 ID;
+    uint32 Flags;
     uint32 Category;
     uint32 Faction;
     uint32 Type;
-    int32 Slot;
```

Comment:
@Shauren @funjoker please check SummonProperties.db2. Slot is NOT an int32, neither uint32. it must be uint8. there are many new spells in legion having Slot = 255 (that is -1 in uint8 format). check the effect handler for summon, it will try to add the Summon into the specified slot IF slot>0. that is key, because m_SummonSlot for players is an array with a capacity of 9. it is unwise to try to add a summon there. it will crash